Sources/Card/CardClient.swift

```diff
@@ -27,31 +25,31 @@ public class CardClient {
     ///   - orderID: The ID of the order to be approved
     ///   - card: The card to be charged for this order
     ///   - completion: Completion handler for approveOrder, which contains data of the order if success, or an error if failure
-    public func approveOrder(orderID: String, card: Card, completion: @escaping (Result<OrderData, Error>) -> Void) {
-        do {
-            let confirmPaymentRequest = try ConfirmPaymentSourceRequest(
-                card: card,
-                orderID: orderID,
-                clientID: config.clientID
-            )
+    public func approveOrder(
+        orderID: String,
+        card: Card,
+        completion: @escaping (Result<OrderData, NetworkingError>, CorrelationID?) -> Void
+    ) {
+        let path = "/v2/checkout/orders/\(orderID)/confirm-payment-source"
+        var request = APIRequest(method: .post, path: path)
+        request.body = ConfirmPaymentSourceBody(card: card)
```

Comment:
Here an `APIRequest` object can be used to provide both the path, HTTP method, and request body; similar to an `HTTPRequest`.